MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,17 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
Why do you use here [NSBundle bundleForClass:] instead of using [NSBundle mainBundle]?
I would prefer keep using mainBundle:
`NSBundle * bundle = [NSBundle mainBundle];`